apps/VideoStream/videoStream.py

```diff
@@ -68,7 +68,7 @@ def send_frame(self, frame, address, quality=4):
         frame = zlib.compress(frame, -1)
         self.socket.sendto(frame, address)
 
-    def listen(self, port=None): #Apparently this is a common workaround. Can't reference self in function arguments.
+    def listen(self, port=None):
         if port is None:
```

Comment:
curious if there will be a case where we will use a port number different than `self.port`?

Reply:
I'm not sure if there would be. When Dragon was helping me to get started he put a default argument there so I kept it in my implementation.